api/app/tests/apps/signals/test_area_model.py

```diff
@@ -0,0 +1,63 @@
+import random
+
+from django.contrib.gis.geos import MultiPolygon, Polygon
+from django.test import TestCase
+
+from signals.apps.signals.models import Area, AreaHistory, AreaType
+
+# TODO: Temporary, will be refactored when approach is deemed sound.
+
+BBOX = [4.58565, 52.03560, 5.31360, 52.48769]
+
+
+def get_random_bbox(bbox=BBOX, n_lon_subdiv=10, n_lat_subdiv=10):
```

Comment:
Duplicated function, why not use the `get_random_bbox` from the factories file? Or maybe move the function to a utils type of file so that it is more reusable.